roles/node/defaults/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 # For ARMv6 models (A, A+, B, B+, Zero, Zero W). This should match a directory name on the download site: https://nodejs.org/dist/ (e.g. latest-v4.x, v5.8.0, latest)
-nodejs_armv6_version: latest-v12.x   # Latest 12.x series LTS release
+nodejs_armv6_version: latest-v10.x   # 10.x is the latest LTS release that supports ARMv6.
```

Comment:
Would it make sense to switch to https://unofficial-builds.nodejs.org/download/release/index.json  ? We must move on from Node 10 pretty soon anyway.

Reply:
Wasn't aware of this, seems like a good idea. 
Unfortunately there is no /latest-v12.x directory, so we would need to hard code the latest version of today, like There is an armv6 version here: https://unofficial-builds.nodejs.org/download/release/v12.19.1/
Which would be hard to maintain. Any suggestions? 

Maybe we should also include a warning when installing signalk-server on an ARMv6 architecture? Something alone the lines of:

Support for ARMv6 architectures (Raspberry Pi Zero, 1 , Compute Module 1) has stopped with the officially supported Node v12 builds. This script is now installing an unofficial, non-supported, version of Node v12. It is suggested to move to a newer supported ARM architecture (Pi 2 and above). See Node support statement: https://github.com/nodejs/unofficial-builds/ 
Additionally, a specific version of Node v12 will be installed, which may be outdated. Please check node/defaults/main.yml for instructions on how to check if a newer updated build is available.

Reply:
Tested it with v12.19.1. The whole process works. Node 12.19.1 is installed. But in the last step it can't start signalk-server:
```
TASK [Restart signalk-server] ******************************************************************************************************************************************************
fatal: [tjielpi]: FAILED! => {"changed": false, "msg": "Unable to restart service signalk-server: Job for signalk-server.service failed because the control process exited with error code.\nSee \"systemctl status signalk-server.service\" and \"journalctl -xe\" for details.\n"}
```
It is looking for /usr/bin/signalk-server while it is actually installed in /usr/local/bin/signalk-server.
This couldn't happen because of the installation of another node version, can it?

Reply:
From a bit of googling: packages managed by a distribution (and i guess package managers as well) go in /usr/bin. Packages you compile/install yourself go in /usr/local/bin. So the unofficial self-installed node v12 for ARMv6 goes in /usr/local/bin (where the official node v12 goes in /usr/bin). Probably that causes signalk-npm to install signalk-server to install in /usr/local as well.
It works correctly if i change signalk.service.j2 to: ExecStart=/usr/local/bin/signalk-server.
So i guess the solution is to include a check signalk-npm to check where signalk-server lives?

Reply:
Fixed and tested. Commit should be part of this PR now.
Additionally updated to latest unofficial Node build v12.20.0